internal/layers/layers.go

```diff
@@ -146,6 +146,19 @@ func MountContainerLayers(ctx context.Context, layerFolders []string, guestRoot
 	hostPath := filepath.Join(layerFolders[len(layerFolders)-1], "sandbox.vhdx")
 	containerScratchPathInUVM := ospath.Join(uvm.OS(), guestRoot)
 	log.G(ctx).WithField("hostPath", hostPath).Debug("mounting scratch VHD")
+
+	// For LCOW we can reuse another containers scratch (generally the sandbox containers)
+	//
+	// It's assumed that we are getting passed a symlink to the sandbox.vhd location on the host for this
+	// functionality, but the below will also work if we just get passed the same sandbox.vhd path
+	// we already have before.
```

Comment:
This comment is a bit confusing. I see what you're saying but it is still ambiguous. Maybe change it to: 

```
For LCOW, we can reuse another container's scratch space (usually the sandbox container's). 

When sharing a scratch space, the `hostPath` will be a symlink to the sandbox.vhd location to use. When not sharing a scratch space, `hostPath` will be the path to the sandbox.vhd to use. 
```

Reply:
Much better

Reply:
Fixed